File: src/main/java/top/dsbbs2/bukkitcord/bungee/BungeeBootstrap.java
package top.dsbbs2.bukkitcord.bungee;

import static top.dsbbs2.bukkitcord.api.PlatformManager.setBootstrapPlugin;

public class BungeeBootstrap extends BungeePluginImpl {
    {
        setBootstrapPlugin(this);
    }
}

File: src/main/java/top/dsbbs2/bukkitcord/bungee/BungeeBootstrap.kt
package top.dsbbs2.bukkitcord.bungee

import top.dsbbs2.bukkitcord.api.PlatformManager.bootstrapPlugin

class BungeeBootstrap : BungeePluginImpl() {
    init {
        bootstrapPlugin = this
        BungeePlatformImpl()
    }
}